roles/pool/src/lib/mining_pool/mod.rs

```diff
@@ -66,6 +66,15 @@ pub struct CoinbaseOutput {
     output_script_value: String,
 }
 
+impl CoinbaseOutput {
```

Comment:
This struct appears all over. I'll create an issue from this comment to consolidate to a location in the code base that makes sense.
https://github.com/stratum-mining/stratum/pull/1066#discussion_r1714014229
https://github.com/stratum-mining/stratum/pull/1066#discussion_r1713911191
https://github.com/stratum-mining/stratum/pull/1066#discussion_r1714003837